idl/internal/y.go

```diff
@@ -1,5 +1,37 @@
 // @generated Code generated by yacc
 
+
+// @generated Code generated by yacc
+
+// Copyright (c) 2019 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
```

Comment:
there's something wrong here. would you mind reverting the changes to this file?

Reply:
I've reset this as well as the lex.go file to the original state